modules/gke/readme.md

```diff
@@ -92,4 +94,4 @@ module "gke" {
 
 Implementing a private cluster has technical requirements, restrictions and limitations. These are outline in [this link](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters#req_res_lim)
 
-* One to be aware of is do not overlap with the range `172.17.0.0/16` as this is an IP range Google uses.
```

Comment:
What a strange diff line (in that I see no difference)

Reply:
My editor probably added a newline to the end of the file.